### PR TITLE
Add required information for migration to Open Rail Association

### PR DESCRIPTION
**Pull request by tongpu**
This PR introduces the files required by the Open Rail Associaton as mentioned in: https://github.com/OpenRailAssociation/technical-committee/issues/118

### CODE REVIEW

**Review by schalbts — COMMENTED** [review]
Hi Lukas
The RCM-DX format is of course intended to be available for railway companies, but also for suppliers and developers of measurement systems.

File: README.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ The RCM-DX format is developed and maintained by the SBB company. An extension o
 
 The RCM-DX format is a file format detailing the HDF5 format version 2.0. HDF5 was chosen for several reasons, including that it is an open format. HDF5 is a hierarchical data storage format where the data is arranged in a tree structure. The HDF5 format is described on the webpage of the HDF5-group, in particular on the site [HDF5 file format specification](https://portal.hdfgroup.org/display/HDF5/). The HDF5 group offers tools and libraries for various programming languages and operating systems that allow to read and write HDF5 files.
 
+The specification was developed within SBB and published as an open source standard in 2022, which allowed for easier collaboration with other railroad companies.
+With the formation of the Open Rail Association it was decided to donate the standard to the association, to put the project under indepentend stewardship.
```
Comment (mxmehl) [inline README.md:23]: Wording suggestion:

```suggestion
The specification was developed within SBB and published as an open source standard in 2022, which allowed for easier collaboration with other railway companies.
With the formation of the OpenRail Association it was decided to donate the standard to the association, to put the project under independent stewardship.
```

Comment (tongpu) [reply README.md]: Wow, I really screwed up independent badly. :cold_sweat:

**Comment by tongpu** [comment]
> The RCM-DX format is of course intended to be available for railway companies, but also for suppliers and developers of measurement systems.

I've mentioned suppliers of measurements systems in the historical context as well.

**Review by mxmehl — APPROVED** [review]
Thanks!

**Review by schalbts — APPROVED** [review]

**Pull request merged**